src/pages-conditional/wallets/index.js

```diff
@@ -247,19 +247,9 @@ const WalletsPage = ({ data }) => {
     })
     .slice(0, 4)
 
-  const hardwareWallets = wallets.filter(
-    (wallet) => wallet.has_hardware === "TRUE"
-  )
-  const whaleWallets = wallets
-    .filter((wallet) => {
-      return (
-        wallet.has_high_volume_purchases === "TRUE" ||
-        wallet.has_limits_protection === "TRUE" ||
-        wallet.has_multisig === "TRUE"
-      )
-    })
-    .slice(0, 4 - hardwareWallets.length)
-  const cryptoConverted = Array.prototype.concat(hardwareWallets, whaleWallets)
+  const cryptoConverted = wallets
```

Comment:
I made this change because it simplifies the logic without impacting any functionality (we have more than 4 hardware wallets, so the `whaleWallets` are never added to the result.

Reply:
Perhaps we change the logic to e.g. grab the first 4 wallets that fit any of this criteria? I do feel like hardware wallets should be most prominent tho.